trade-finance-manager-api/src/v1/api.js

```diff
@@ -759,6 +759,35 @@ const updateACBSfacility = async (facility, deal) => {
   return {};
 };
 
+/**
+ * ACBS facility amendment
+ * @param {String} ukefFacilityId UKEF Facility ID
+ * @param {Object} amendments Facility object comprising of amendments
+ * @returns {Object} updated FMR upon success otherwise error
+ */
+const amendACBSfacility = async (ukefFacilityId, amendments, deal) => {
+  if (amendments) {
+    try {
+      const response = await axios({
+        method: 'post',
+        url: `${refDataUrl}/acbs/facility/${ukefFacilityId}/amend`,
```

Comment:
the general convention that we used for endpoints was to use the full name of it
i.e. `.../amendment` - if we are changing 1 amendment 
or
`../amendments` if we we are expecting more items

Reply:
second item:
are we using the `ukefFacilityId` in the endpoint? shouldn't we use the `_id` (MongoDB) item instead?

Reply:
1. Good spot, have updated.
2. Nope `ukefFacilityId` to keep it consistent with Azure DF calls and also I need `ukefFacilityId` to extract FMR which can only be fetched (GET) using `ukefFacilityId`.